emulator/index.js

```diff
@@ -0,0 +1,32 @@
+// Require Mock Mobile
+require('../thali/mockmobile');
+var ThaliReplicationManager = require('../thali/thalireplicationmanager');
```

Comment:
Why doesn't this jx install thali like we tell people to do when they use the plugin normally? I'm only picking nits because it's good to have the test environment be as close to production as possible.